replimanWindow.cpp

```diff
@@ -472,8 +505,13 @@ replimanWindow::MessengerForTarget(type_code w) const
 			// The Desktop is owned by Tracker and the Shelf is in the
 			// View "PoseView" in Window "Desktop"
 			request.AddSpecifier("View", "PoseView");
-			request.AddSpecifier("Window", "/boot/home/Desktop");
-			to = BMessenger("application/x-vnd.Be-TRAK", -1);
+			if (FindDesktopMessenger(&to) == false)
+			{
+				// Window 0 works also fine,
+				// but it is undocumented feature that may be changed in the future
+				request.AddSpecifier("Window", 0);
```

Comment:
There's an issue here on 32bit:

> /Source/Repliman/replimanWindow.cpp: In method `class BMessenger replimanWindow::MessengerForTarget(long unsigned int) const':
> /Source/Repliman/replimanWindow.cpp:512: call of overloaded `AddSpecifier(const char[7], int)' is ambiguous
> /boot/system/develop/headers/os/app/Message.h:121: candidates are: status_t BMessage::AddSpecifier(const char *, long int)
> /boot/system/develop/headers/os/app/Message.h:125:                 status_t BMessage::AddSpecifier(const char *, const char *)
> 

Solution:
```suggestion
				request.AddSpecifier("Window", int32(0));
```

Reply:
@humdingerb fixed